tests/test_examplecases.py

```diff
@@ -30,18 +30,19 @@ def test_few_steps_RCM():
     Test running a few steps
     '''
     # defining / initializing
-    particle = Particle(params) # define the particle
+    particle = Particles(params) # define the particle
     np.random.seed(0) # fix the random seed
-    all_walk_data = None # init the walk data
+    all_walk_data = particle.generate_particles(Np_tracer, seed_xloc,
+                                                seed_yloc) # init the walk data
```

Comment:
Might want to search for calls to `generate_particles` that expect a `walk_data` output, this might show up in a few places